doc/Analytics.xml

```diff
@@ -3397,127 +3397,71 @@ Added Recognition rule engines</revremark>
     </section>
     <section>
       <title>Counting rule</title>
-      <para>Counting rule counts the number of motion object passing through the set of line segments (barrier) and optional direction attribute. The configuration parameters also include the time interval to report the events and time interval to reset its counter. </para>
+      <para>Counting rule counts the number of objects passing through a set of line segments
+        (barriers) with an optional direction. The configuration parameters also include the time
+        interval to report the events and time interval to reset its counter.<variablelist>
+          <varlistentry>
+            <term>Parameters</term>
+            <listitem>
+              <para role="param">Segments [tt:Polyline]</para>
+              <para role="text">One or more polylines used for detection. An object must have passed
+                through all of them to register as a count. ??? or ??? Passing through any of the
+                lines registers as a count. ???</para>
+              <para>ReportTimeInterval [xs:duration]</para>
```

Comment:
Purpose of ReportTimeInterval unclear: The event is a property event and hence there is no need for repetition. Only reason I can imagine is to reduce the number of events in case of frequent updates. Propose to drop

Reply:
That's true. I think we should make it a non-property event because I think there are use cases for having an periodical update, for statistical purposes. It's another use case of getting event for every object or every n:th object. Maybe we should have two different events.

Reply:
Don't get the benefit of a non-property event. With a property event the client always knows the last value. If the client wants to do statistics it should just subscribe and then do whatever it needs to create its statistics.

Reply:
Sorry see question raised above.

Reply:
Prefer to keep ReportingTimeInterval as optional parameter